tools/edta/.shed.yml

```diff
@@ -0,0 +1,11 @@
+categories:
+    - Variant Analysis
+description: |
+    The EDTA package was designed to filter out false discoveries in raw TE candidates and generate a high-quality non-redundant TE library for whole-genome TE annotations. 
+long_description: |
+    The EDTA package was designed to filter out false discoveries in raw TE candidates and generate a high-quality non-redundant TE library for whole-genome TE annotations. Selection of initial search programs were based on benckmarkings on the annotation performance using a manually curated TE library in the rice genome.
+name: edta
+owner: iuc
```

Comment:
you are not on IUC here, so the owner would be "bgruening"

Reply:
Thanks for all your feedback @bgruening!

Reply:
Hi @bgruening, the checks have passed. Can you help to merge to the main branch? Thanks!

Reply:
@tcollins2011 I longed you yesterday about the open comments above. Do you want to fix them as well?